.github/workflows/repotests.yml

```diff
@@ -554,6 +555,33 @@ jobs:
           bin/cdxgen.js -p -r -t dotnet-framework repotests/SimpleFrameworkApp -o bomresults/bom-dotnet-framework.json
           bin/cdxgen.js -p -r -t dotnet-framework repotests/Reporting-Windows-Application -o bomresults/bom-dotnet-framework-reporting.json --deep
         shell: bash
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+      - uses: sbt/setup-sbt@v1
+      - uses: coursier/setup-action@v1
+        with:
+          apps: scala3 scalac
+      - name: scala tests
+        run: |
+          python -m pip install atom-tools
+          npm install -g @appthreat/atom
```

Comment:
While cdxgen is able to invoke atom via the local node_modules directory, atom is trying to invoke `scalasem` from the PATH. In a separate PR, we need to enhance cdxgen the set up the path to the local node_modules/.bin directory while doing `executeAtom`.